deeds-dapp-service/src/main/java/io/meeds/deeds/service/MeedTokenMetricService.java

```diff
@@ -64,7 +64,8 @@ public class MeedTokenMetricService {
   private MeedTokenMetricsRepository meedTokenMetricsRepository;
 
   @Getter
-  private MeedTokenMetric            recentMetric;
+  @Setter
+  protected MeedTokenMetric            recentMetric;
```

Comment:
why changing this to `protected` instead of `private` knowing that you already have getter and setter on that field ?

Reply:
because the setter doesn't appear (to use it) when recentMetric is private

Reply:
It must appear, you may not having `lombok` in you IDE